### DISCUSSION TITLE
Baldi's basics mod menu may contain malware.

### DISCUSSION THREAD

**Discussion by EpicMario5345** [community/community#37426] [Other Feature Feedback, Questions, & Ideas]
So, I don't know if this was said before, but scanning the Android versions of your mod menus for Baldi's Basics in VirusTotal came up with some surprising results.
Older versions has adware and password loggers, while the most recent version has literal malware.

I need confirmation that this isn't the case.

**Comment by ossanna16** [comment]
Hi @EpicMario5345,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on the specific topic. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
